### PR TITLE
[RW-6717][risk=no]React Routes on Home View

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 1]
```diff
@@ -284,7 +324,7 @@ export const dBHomeComponent = (
                                 <span className='icon-link'>FAQs</span>
                             </a>
                         </div>
-                        <div className='icons' onClick={() => this.iconClickEvent('Intro-Videos')}>
+                        <div className='icons' onClick={() => this.iconClickEvent('introductory-videos')}>
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:327]: Navigation

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 2]
```diff
@@ -147,19 +147,61 @@ const styles = reactStyles({
...
+                    const url = 'survey/' + info.name.replace(' ', '-').toLowerCase();
+                    NavStore.navigateByUrl(url);
+                    break;
+            }
+        }
+    }
+    render() {
+        const { name, description, questionCount, standardConceptCount, participantCount } = this.props;
+        return <div
+            onClick={() => this.resultClick(this.props)}
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:203]: Navigaion

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 3]
```diff
@@ -147,19 +147,61 @@ const styles = reactStyles({
...
+                    NavStore.navigateByUrl('ehr/drug-exposures');
+                    break;
+                // MEASUREMENT
+                case 21:
+                    NavStore.navigateByUrl('ehr/labs-and-measurements');
+                    break;
+                // PROCEDURE
+                case 10:
+                    NavStore.navigateByUrl('ehr/procedures');
+                    break;
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:185]: Navigation in function

File: public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx [hunk 4]
```diff
@@ -216,8 +257,7 @@ export const dBHomeComponent = (
             this.search(val);
         }
         iconClickEvent(iconString: string) {
-            // dbc.triggerEvent('HelpEvent', 'Help', 'Click',
-            //     iconString, null, null);
+            if (iconString === 'introductory-videos') { NavStore.navigateByUrl('/' + iconString); }
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx:260]: Navigation in function

**Review by dolbeew — APPROVED** [review]

**Pull request merged**